modules/ROOT/pages/bat-install-task.adoc

```diff
@@ -5,10 +5,28 @@ endif::[]
 
 The Blackbox Automated Testing (BAT) CLI wrapper is a command line wrapper that you install to execute API tests.
 
-== Prerequisite
+== Prerequisite for Linux and MacOS Systems
 
 Before installing the BAT CLI, ensure that AdoptOpenJDK's OpenJDK 8 is installed on your system.
 
+== Prerequisites for Windows 10 Systems
+
+. Install Windows Subsystem Linux (WSL)
++
+The instructions are at https://docs.microsoft.com/en-us/windows/wsl/install-win10.
+. Install Java 8, cURL, and unzip in the Linux distribution that you installed in your WSL.
```

Comment:
```suggestion
. Install Java 8, install cURL, and unzip the Linux distribution that you installed in your WSL.
```
OR do you mean this:
```suggestion
. Unzip the Linux distribution that you installed in your WSL and install into it Java 8 and cURL.
```

Reply:
Hi, the required tools are:
- Java 8
- curl
- unzip (this is the name of the cli tool that unzips .zip files in linux)
The 3 of them must be installed in the WSL distro.